spec/league_statable_spec.rb

```diff
@@ -116,11 +116,7 @@
 
     describe '#count_of_teams' do
         it 'returns the count of teams from the game_teams.csv file' do
-            game_teams_path = './data/game_teams.csv'
-
-            unique_team_count = LeagueStatable.count_of_teams(game_teams_path)
```

Comment:
You should be able to access these methods from stat_tracker

Reply:
Those are the old tests. I no longer have those in the code.
should only include the 'expect(@stat_tracker.count_of_teams).to eq 32'